assets/js/protocol/protocol-navigation.js

```diff
@@ -191,16 +231,16 @@ if (typeof window.Mzp === 'undefined') { // eslint-disable-line block-scoped-var
     Navigation.bindEvents = function() {
         _navItemsLists = document.querySelectorAll('.mzp-c-navigation-items');
         if (_navItemsLists.length > 0) {
-            var navButtons = document.querySelectorAll('.mzp-c-navigation-menu-button');
-            for (var i = 0; i < navButtons.length; i++) {
-                navButtons[i].addEventListener('click', Navigation.onClick, false);
-            }
+            _navMenuButtons = document.querySelectorAll('.mzp-c-navigation-menu-button');
+            _navMenuButtons.forEach(function (menuButton) {
+                menuButton.addEventListener('click', Navigation.onClick, false);
+            });
             Navigation.setAria();
         }
     };
 
     /**
-     * Initialise menu.
+     * Initialize menu.
```

Comment:
🇺🇸

Reply:
is it! oops - it just showed up on the spell checker 😿

Reply:
Sorry, I just meant to tease for the change from 🇨🇦 to 🇺🇸 . It's a good change.